reference/service/webhooks.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Pulumi Webhooks
+---
+
+Webhooks allow you to build applications that will be notified when events
+happen on Pulumi. For example, whenever a stack is updated, an team is modified,
+and so on. Whenever an event occurres will send an HTTP `POST` request to all
```

Comment:
Typo, "occurres" => "occurs."

Reply:
Should this also be, `Whenever an event occurs, Pulumi will..`?

Reply:
Done.